tests/napkin_test.ml

```diff
@@ -1,207 +0,0 @@
-module IO = Res_io
```

Comment:
@IwanKaramazow this file is a duplicate of `res_test.ml` and isn't used?

Reply:
Remove if unused. Probably not converted